### ISSUE TITLE
k8s集群部署插件不会同步吗

### ISSUE THREAD

**Issue by hlovingness** [msgbyte/tailchat#188]
k8s 中启了两个pod，两个pod在不通机器上，我在面板上安装了插件，但是到了另外一台电脑上插件就不同步了，需要重新安装。

**Comment by moonrailgun** [comment]
跟k8s没有关系。
插件分为前端插件、后端插件。
前端插件是跟设备走的。用户可以自由安装。拥有的能力也就是前端能力

**Comment by hlovingness** [comment]
我同一个账号在公司电脑登录和在家里电脑登录那这个插件要重新安装了

**Comment by moonrailgun** [comment]
yes, because of plugin can apply in login view, which not logined
